_data/betting.yml

```diff
@@ -19,3 +19,11 @@ websites:
     twitter: win2dayat
     facebook: win2dayat
     lang: de
+    
+  - name: Wolf.bet
+    url: https://wolf.bet/
+    img: wolfbet.png
+    tfa:
+      - totp
+      - email
+    doc: https://wolf.bet/user/settings?tab=2fa
```

Comment:
The doc link requires the user to be logged in, so it's not publicly available. They do however have a YouTube video regarding 2FA:
```suggestion
    doc: https://www.youtube.com/watch?v=CgAWzFIwNOk
```

There is no mention of email 2FA. Do you have any more information about that? Can you provide screenshots of the setup process? (With all personal information redacted, of course.)

Also, the icon file size is larger than necessary. Please replace it with this one:
![wolfbet](https://user-images.githubusercontent.com/520841/93642002-9fb98880-f9fd-11ea-83e3-4c1c24cec994.png)

Reply:
fixed